message Test1Msg {
  required string stringType = 1;
  required double doubleType = 2;
  optional Test1Msg nestedType = 3;
}

message Test2Msg {
  required string stringType = 1;
  required double doubleType = 2;
  optional Test1Msg nestedType = 3;
}
